### PR TITLE
Service: Label Studio

### PR DESCRIPTION
**Pull request by rennokki**
## Changes
- Added [LabelStudio](https://labelstud.io/guide/get_started)

## Issues
N/A

### CODE REVIEW

**Comment by rennokki** [comment]
@andrasbacsai

File: templates/compose/labelstudio.yaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
+# documentation: https://labelstud.io/guide/
+# slogan: Label Studio is a multi-type data labeling and annotation tool with standardized output format
+# tags: workflow, orchestration, data-pipeline, python, labelstudio, ai, elasticsearch, datasets, data, machine-learning, data-science, nlp, images, vision
+# logo: svgs/labelstudio.png
+# port: 8080
+
+version: "3"
```
Comment (peaklabs-dev) [inline templates/compose/labelstudio.yaml:7]: Only for future reference, no longer needed for compose projects

**Comment by peaklabs-dev** [comment]
I have tested this service and it seems to work properly. Thanks for your PR 💜.

File: templates/compose/labelstudio.yaml (hunk 1 above)
Comment (rennokki) [reply templates/compose/labelstudio.yaml:7]: Got it. Gonna modify all of them.

**Pull request merged**